components/pd_client/src/client.rs

```diff
@@ -698,3 +698,21 @@ impl PdClient for RpcClient {
             .execute()
     }
 }
+
+pub struct DummyPdClient {
+    pub next_ts: TimeStamp,
+}
+
+impl DummyPdClient {
+    pub fn new() -> DummyPdClient {
+        DummyPdClient {
+            next_ts: TimeStamp::zero(),
+        }
+    }
+}
+
+impl PdClient for DummyPdClient {
+    fn get_tso(&self) -> PdFuture<TimeStamp> {
+        Box::new(future::ok(self.next_ts))
```

Comment:
should we increase the inner ts to make it more like a real TSO?

Reply:
We can if we need to in the future; for now we don't need it.